src/chocolatey.console/Program.cs

```diff
@@ -124,6 +126,8 @@ private static void Main(string[] args)
                     }
                 }
 
+                check_installed_dotnetfx_version();

```

Comment:
Is there a reason not to do this as the very first thing that is done in this method?

Reply:
If I do it at the beginning of the method, logging it not set up, so it does not output anything to the screen or to the logs.

Reply:
Ah, gotcha. Makes sense.